packages/dd-trace/test/config.spec.js

```diff
@@ -1299,7 +1298,7 @@ describe('Config', () => {
     expect(config).to.have.nested.property('appsec.eventTracking.enabled', true)
     expect(config).to.have.nested.property('appsec.eventTracking.mode', 'safe')
     expect(config).to.have.nested.property('appsec.apiSecurity.enabled', true)
-    expect(config).to.have.nested.property('appsec.apiSecurity.requestSampling', 1.0)
+    expect(config).to.have.nested.property('appsec.apiSecurity.sampleDelay', 10)
```

Comment:
no more options, no need to check it anymore

Reply:
Done